Makefile.amigaos_GCC

```diff
@@ -0,0 +1,121 @@
+##########################################################################
```

Comment:
I  don't believe we need one more makefile. Maybe there is a way to make it work in the current amigaos makefile

Reply:
I didn't want to touch original VBCC makefile, so I cloned/modified to GCC